src/utils/PasswordScorer.js

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import Zxcvbn from 'zxcvbn';
+import zxcvbn from 'zxcvbn';
```

Comment:
Does this actually work still?

Reply:
Yes, confirmed to work. When you import the default export like this, the importer gets to pick the name.